src/sentry/auth/elevated_mode.py

```diff
@@ -1,38 +1,47 @@
 from abc import ABC, abstractmethod
+from enum import Enum
+from typing import Tuple
+
+
+class InactiveReason(str, Enum):
+    INVALID_IP = "invalid-ip"
+    INCOMPLETE_SSO = "incomplete-sso"
+    # Indicates the request should be allowed
+    NONE = None
+
+    def __bool__(self):
+        return self.value is not None
```

Comment:
We use the enum in a truthy check, InactiveReason.NONE should be falsey

Reply:
here right? https://github.com/getsentry/sentry/blob/f906a6cc4686609360aba7ab83b0947e7ac8c3e2/src/sentry/auth/superuser.py#L282

Reply:
Yup that's the one